plugins/fclrave/fclcollision.cpp

```diff
@@ -1217,20 +1216,23 @@ std::pair<FCLSpace::FCLKinBodyInfo::LinkInfo*, LinkConstPtr> FCLCollisionChecker
     return std::make_pair(link_raw, LinkConstPtr());
 }
 
-std::pair<FCLSpace::FCLKinBodyInfo::FCLGeometryInfo*, GeometryConstPtr> FCLCollisionChecker::GetCollisionGeometry(const fcl::CollisionObject &collObj)
+void FCLCollisionChecker::GetCollisionGeometryAndName(const fcl::CollisionObject &collObj, GeometryConstPtr& pgeom, std::string& geometryName)
 {
     const std::shared_ptr<const fcl::CollisionGeometry>& collgeom = collObj.collisionGeometry();
     FCLSpace::FCLKinBodyInfo::FCLGeometryInfo* geom_raw = static_cast<FCLSpace::FCLKinBodyInfo::FCLGeometryInfo *>(collgeom->getUserData());
```

Comment:
let's use this opportunity to unify naming, please change `geom_raw` to `geomRaw`